test/integration/main_test.go

```diff
@@ -102,7 +133,7 @@ func restartInsightsOperator(t *testing.T) {
 
 	for _, pod := range pods.Items {
 		clientset.CoreV1().Pods("openshift-insights").Delete(pod.Name, &metav1.DeleteOptions{})
-		err := wait.PollImmediate(1*time.Second, 20*time.Minute, func() (bool, error) {
+		err := wait.PollImmediate(1*time.Second, 1*time.Minute, func() (bool, error) {
```

Comment:
I don't think pods are usually deleted within 1 minute, I'd give it more time

Reply:
Actually 1 minute is with quite a reserve. Normally it takes few seconds and Kubernetes is making sure that deletion wont be longer then Grace period, which is 30s in IO pod.
You can see moure about it here: https://cloud.google.com/blog/products/gcp/kubernetes-best-practices-terminating-with-grace